src/GitHub.Exports.Reactive/ViewModels/IPullRequestCreationViewModel.cs

```diff
@@ -1,7 +1,10 @@
-﻿namespace GitHub.ViewModels
+﻿using ReactiveUI;
+
+namespace GitHub.ViewModels
 {
     [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1040:AvoidEmptyInterfaces")]
     public interface IPullRequestCreationViewModel : IViewModel
     {
+        ReactiveCommand<object> CancelCommand { get; }
```

Comment:
Seeing as this is going to be more useful in the future, how about we put this in its own interface (something like `IReactiveViewModel` that inherits from `IViewModel`) and make `BaseViewModel` implement that? We can add an accessor for it in `SimpleViewUserControl`.